mod.yaml

```diff
@@ -3,12 +3,12 @@
 Metadata:
 	Title: Over Powered Mod
 	Description: Crazy modifications to Red Alert by Boolbada
-	Version: 007-Playtest-20170304
+	Version: {DEV_VERSION}
 	Author: Boolbada
 
 RequiresMods:
-	#modchooser: playtest-20170304
 	modchooser: {DEV_VERSION}
+	as: {DEV_VERSION}
```

Comment:
No, and I completely object the idea of a minimal mod.yaml even. The Mods.AS project as it is at the moment, is supposed to be a plugin using the plugin notation and not a dummy mod. That's how it got set up in CD, SP and NG. Unless you changed any of those as well.